framework/src/st_things/things_stack/utils/things_rtos_util.c

```diff
@@ -44,6 +44,7 @@ static const thread_info_s things_stack_thread_name[THINGS_STACK_MAX_INDEX] = {
 	{"THINGS_STACK_WIFI_JOIN", 8 * 1024},		/*THINGS_STACK_WIFI_JOIN_THREAD */
 	{"THINGS_STACK_FOTA_UPDATE", 6 * 1024},	/*THINGS_STACK_FOTA_THREAD */
 	{"THINGS_STACK_AP_INFO_SET", 4 * 1024},/*THINGS_STACK_AP_INFO_SET_THREAD */
+	{"THINGS_STACK_AP_SCAN_THREAD", 4 * 1024},/*THINGS_STACK_AP_SCAN_THREAD */
```

Comment:
did you check heapinfo?
i think it's too much stack size.

Reply:
How we check, what is the min stack size we have to take for thread,  means which parameter we have to check for stack size.

Reply:
@Hyunjun85  I used heapinfo command.
I was showing memory stack : 4112 , curr_heap :0, peak_heap:4112

Reply:
Check with thread stack size is 1024
 the PEAK_STACK size is going till 612.
 So I take thread stack size is 1024